### PR TITLE
Create Clay-Navbar

### PR DESCRIPTION
**Pull request by carloslancha**

### CODE REVIEW

**Comment by carloslancha** [comment]
This is just one part of the component. Waiting for clayDropdown to be develop to continue.

**Comment by carloslancha** [comment]
ci:retest

**Comment by carloslancha** [comment]
Depends on https://github.com/metal/metal-clay-components/pull/75

File: packages/clay-navbar/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	output: {
+		library: 'metal',
+		libraryTarget: 'this',
+		filename: './build/globals/clay-navbar.js',
+	},
+	plugins: [new webpack.optimize.ModuleConcatenationPlugin()],
+	resolve: {
+		mainFields: ['esnext:main', 'main'],
+		alias: {
+			'incremental-dom': path.resolve('./node_modules/incremental-dom'),
```
Comment (jbalsas) [inline packages/clay-navbar/webpack.config.js]: We need to add `metal-soy` as in https://github.com/metal/metal-clay-components/pull/76

File: packages/clay-navbar/src/ClayNavbar.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+	inverted: Config.bool().value(false),
+
+	/**
+	 * List of pages to show in the Navbar.
+	 * @instance
+	 * @memberof ClayNavbar
+	 * @type {!Array}
+	 * @default undefined
+	 */
+	pages: Config.arrayOf(
```
Comment (jbalsas) [inline packages/clay-navbar/src/ClayNavbar.js]: Why pages? What if they are not pages? How about simple `navItems` or just `items`?

File: packages/clay-navbar/src/ClayNavbar.soy [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+			{if $items}
+				{call .navmenu}
+					{param items: $items /}
+				{/call}
+			{/if}
+		</div>
+	</nav>
+{/template}
+
+{template .navitem}
```
Comment (jbalsas) [inline packages/clay-navbar/src/ClayNavbar.soy]: Missing comment

File: packages/clay-navbar/src/ClayNavbar.soy [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+			{param href: $url /}
+			{param label kind="html"}
+				<span class="navbar-text-truncate">{$title}</span>
+			{/param}
+
+		{/call}
+	</li>
+{/template}
+
+{template .navmenu}
```
Comment (jbalsas) [inline packages/clay-navbar/src/ClayNavbar.soy]: Missing comment

**Pull request merged**